turtleFSI/modules/domain.py

```diff
@@ -7,47 +7,85 @@
 from dolfin import ds, MPI
 
 """
-Date: 2022-11-07
-Comment from Kei: Naming here is a bit confusing. For example, dx_f, ds_s are hard to understand. 
-dx refers to the area of the domain while ds refers to the area of the boundary.
-f refere to the fluid while s refers to the solid.
+Last update: 2023-06-12
+Kei Yamamoto added docstring for assign_domain_properties function and added comments in the function.
 """
 
 def assign_domain_properties(dx, dx_f_id, rho_f, mu_f, fluid_properties, dx_s_id, material_model, rho_s, mu_s, lambda_s, solid_properties, domains, ds_s_id, boundaries, robin_bc, **namespace):
     """
     Assigns solid and fluid properties to each region.   
+
```

Comment:
Is there a reason for not using typing of the input arguments, instead of typing in the doc-strings?

Reply:
Do you mean writing, for example, `def assign_domain_properties(dx_f_id : str or list)`?

Reply:
I do not really have any reason for that. I can fix this if typing of the input arguments is preferred 👍

Reply:
If you do it in that way, one can use modules such as [mypy](https://mypy-lang.org/) for checking the types of input arguments.
This was a feature added in Python 3.5, see for instance: https://docs.python.org/3.8/library/typing.html

Reply:
Okay, so the point of using typing of the input arguments is to automatically check the type of input arguments?

Reply:
Yes. It is also easy to forget to update the doc-strings everywhere, and when doing it with strings, there is no way of checking for a consistent syntax

Reply:
That makes sense! Thanks for the explanation. I will update accordingly.

Reply:
You can be Even more specific that saying list, like
`List[dolfin.Function]`
or `Dict[key_type, value_type]`
Its up to you how detailed you'd like to be

Reply:
Thanks for the comment! I think current version has enough detail for now.